README.md

```diff
@@ -0,0 +1,56 @@
+> NOTE: We have recently made significant changes to our repository structure. In order to 
+streamline our development process and foster better contributions, we have merged three separate 
+repositories Cumulus, Substrate and Polkadot into this repository. Read more about the changes [
+here](https://polkadot-public.notion.site/Polkadot-SDK-FAQ-fbc4cecc2c46443fb37b9eeec2f0d85f).
+
+# Polkadot SDK
+
+![](https://cms.polkadot.network/content/images/2021/06/1-xPcVR_fkITd0ssKBvJ3GMw.png)
+
+[![StackExchange](https://img.shields.io/badge/StackExchange-Community%20&%20Support-222222?logo=stackexchange)](https://substrate.stackexchange.com/)
+
+The Polkadot SDK repository provides all the resources needed to start building on the Polkadot 
+network, a multi-chain blockchain platform that enables different blockchains to interoperate and 
+share information in a secure and scalable way. The Polkadot SDK comprises three main pieces of 
+software:
+
+## [Polkadot](./polkadot/)
+[![PolkadotForum](https://img.shields.io/badge/Polkadot_Forum-e6007a?logo=polkadot)](https://forum.polkadot.network/) [![Polkadot-license](https://img.shields.io/badge/License-GPL3-blue)](./polkadot/LICENSE)
+
+Implementation of a https://polkadot.network node in Rust based on the Substrate framework. This 
+directory contains runtimes for the Polkadot, Kusama, and Westend networks. 
```

Comment:
Maybe link to the runtimes repo here, as the Polkadot and Kusama runtimes will be moved out if this repo soon.  
Then only Westend and Rococo remain here.

Reply:
done